assets/js/common/ActivityLogOverview/ActivityLogOverview.jsx

```diff
@@ -56,7 +56,7 @@ function ActivityLogOverview({
   const [selectedEntry, setEntry] = useState({});
 
   const activityLogTableConfig = {
-    pagination: true,
+    pagination: false,
```

Comment:
**nitpick:** In order to disable a table's pagination it is sufficient not opting in for it
```suggestion
```
---
**thought:** as a developer I would find it useful being able to instrument the table component to use either a page based or cursor based pagination.‏
ie:
```js
const tableConfig: {
    pagination: true, // <-- If I want the page based pagination
    cursorPagination: true // <-- If I want the cursor based pagination
}
```
This might be overly simplistic as we might need further configuration based on the chosen pagination (ie what to do on page selection or what to do on prev/next click) so we could consider options like:
```js
const tableConfig: {
    pagination: {
        onThis: () => {},
        onThat: () => {},
    },
    cursorPagination: {
        onThis: () => {},
        onThat: () => {},
    },
}

// or even
const tableConfig: {
    pagination: {
        type: 'page_based' // or 'cursor_based' 
        onSomethingValidOnlyForPageBasedPagination: () => {},
        onSomethingValidOnlyForCursorBasedPagination: () => {}
    }
}
```
Currently if we want a table with page based pagination we can configure the table, otherwise if we want a table with cursor based pagination we need a different approach and I see an opportunity to improve semantics and consistency. In whatever direction.

I am aware that _currently_ tables do not strictly necessarily need cursor based pagination, except for the activity log one, so I'd be fine with this. Let's see how it evolves.

Reply:
The difference in pagination is not numbered/cursor but controlled/uncontrolled.

The pagination in `<Table>` uses the local state of the component. The kind of pagination we need in activity log, instead, needs to be controlled by the page component, as every changes involves a HTTP request. 

Given that 99% of our tables use local data, I opted for the following architecture:
```
IFyou need numbered&uncontrolled pagination
THEN you can use build-in pagination
ELSE you must override it with a custom implementation
```

Should we aim for a _do-it-all_ `<Table>` component?

I think providing the `<Table>` component with both controlled and uncontrolled pagination (combined with numbered/cursor) will lead to an overengineered `<Table>` component. We may prefer composition to inheritance/parametrization, hence if we don't like the logic above I praise for Separating the pagination from the table rendering instead of stuffing everything into one component.

We may want to refactor the component, though. IMHO we should do it in a dedicated PR. We can:
1. block this PR until such refactor
2. merge this PR, then refactor
3. decide we don't need such refactor

Thoughts?

Reply:
I understand the rationale about 99% of our tables use local data, and as mentioned I am fine with the proposal :smile: 

No, let's not block this PR and let's merge it.
I am coming from a consistency/semantic stance, but I understand the reasons and I can live with this currently and see how it evolves.